stdlib/builtin/symbol.rbs

```diff
@@ -36,99 +34,202 @@ class Symbol < Object
-  # Returns whether *sym* is :“” or not.
+  # Returns whether *sym* is :"" or not.
+  # 
   def empty?: () -> bool
 
-  # Returns the [Encoding](https://ruby-doc.org/core-2.6.3/Encoding.html)
-  # object that represents the encoding of *sym* .
+  # Returns the Encoding object that represents the encoding of *sym*.
+  # 
   def encoding: () -> Encoding
 
-  # Returns the name or string corresponding to *sym* .
+  # Returns true if `sym` ends with one of the `suffixes` given.
+  # 
+  #     :hello.end_with?("ello")               #=> true
+  # 
+  #     # returns true if one of the +suffixes+ matches.
+  #     :hello.end_with?("heaven", "ello")     #=> true
+  #     :hello.end_with?("heaven", "paradise") #=> false
+  # 
+  def end_with?: (String str, *String str) -> bool
+
+  # Returns the name or string corresponding to *sym*.
+  # 
+  #     :fred.id2name   #=> "fred"
+  #     :ginger.to_s    #=> "ginger"
   # 
-  # ```ruby
-  # :fred.id2name   #=> "fred"
-  # :ginger.to_s    #=> "ginger"
-  # ```
   def id2name: () -> String
 
   # Returns the representation of *sym* as a symbol literal.
   # 
-  # ```ruby
-  # :fred.inspect   #=> ":fred"
-  # ```
+  #     :fred.inspect   #=> ":fred"
+  # 
   def inspect: () -> String
 
-  # In general, `to_sym` returns the `Symbol` corresponding to an object. As
-  # *sym* is already a symbol, `self` is returned in this case.
+  # In general, `to_sym` returns the Symbol corresponding to an object. As *sym*
+  # is already a symbol, `self` is returned in this case.
+  # 
   def intern: () -> self
 
-  # Same as `sym.to_s.length` .
+  # Same as `sym.to_s.length`.
+  # 
   def length: () -> Integer
 
-  def match: (untyped obj) -> Integer?
+  # Returns `sym.to_s.match`.
+  # 
+  def match: (Regexp | String pattern, ?Integer pos) ?{ (MatchData) -> void } -> MatchData?
 
-  def match?: (*untyped args) -> untyped
+  # Returns `sym.to_s.match?`.
+  # 
+  def match?: (Regexp | String pattern, ?Integer pos) -> bool
 
-  # Same as `sym.to_s.succ.intern` .
+  # Same as `sym.to_s.succ.intern`.
+  # 
   def next: () -> Symbol
 
-  # Same as `sym.to_s.succ.intern` .
-  def succ: () -> Symbol
+  # Same as `sym.to_s.length`.
+  # 
+  alias size length
 
-  # Same as `sym.to_s.swapcase.intern` .
-  def swapcase: () -> Symbol
+  # Returns `sym.to_s[]`.
+  # 
+  alias slice `[]`
 
-  # Returns a *Proc* object which responds to the given method by *sym* .
+  # Returns true if `sym` starts with one of the `prefixes` given. Each of the
+  # `prefixes` should be a String or a Regexp.
   # 
-  # ```ruby
-  # (1..3).collect(&:to_s)  #=> ["1", "2", "3"]
-  # ```
-  def to_proc: () -> Proc
+  #     :hello.start_with?("hell")               #=> true
+  #     :hello.start_with?(/H/i)                 #=> true
+  # 
+  #     # returns true if one of the prefixes matches.
+  #     :hello.start_with?("heaven", "hell")     #=> true
+  #     :hello.start_with?("heaven", "paradise") #=> false
+  # 
+  # variadic type parameter is not supported yet
```

Comment:
What does this comment mean?? @hanachin 
I think the type is the best one for this method.

Reply:
Woops, I forget to delete that.
First I wrote `def start_with?: (String str) -> bool` then realize this method take more than one arguments, but I didn't get how to write variadic arguments at the moment. Finally I got it but forgot to :scissors: the comments.